### DISCUSSION TITLE
Project config support

### DISCUSSION THREAD

**Discussion by nstcactus** [engram-design/FeedMe#901] [Ideas]
### Description

Feed me configuration isn't handled by the project config mecanism.
Is adding support for project config planned in the near future or is it by design?

### Additional info

- Plugin version: 4.1.1
- Craft version: 3.1.33

**Comment by mikesnoeren** [comment]
Any update on this?

**Comment by ThomasDeMarez** [comment]
This would be a welcome addition as now mappings can get lost when updating a feed without any warning. With the implementation of project config we could pick up on those lost mappings in version control.
